src/components/molecules/NavBar/NavBar.scss

```diff
@@ -7,6 +7,7 @@ $border-radius: 28px;
   display: flex;
   justify-content: space-between;
   align-items: center;
+  background-color: inherit;
```

Comment:
@sunny-viktoryia How is this intended to differ from the `background: $black` set on line 6? I believe both of these properties are going to try and set the background color, so one of them is presumably wrong/should be removed.

Reply:
ooops, was so happy that my fix worked, that forgot to check where the property is not needed. :see_no_evil:  
fixed: https://github.com/sparkletown/sparkle/pull/1350/commits/b6e481c7aad3433f43e567cdd0244c43fccfc78f

Reply:
Haha, I know that feel.